js/main.js

```diff
@@ -568,12 +568,33 @@ const MM = (function () {
 		 */
 		modulesStarted: function (moduleObjects) {
 			modules = [];
+			let startUp = "";
+
 			moduleObjects.forEach((module) => modules.push(module));
 
 			Log.info("All modules started!");
 			sendNotification("ALL_MODULES_STARTED");
```

Comment:
I no longer thought about “ALL_MODULES_STARTED”

it makes more sense ;)

Reply:
@bugsounet is this an "o.k." or a change request?

Reply:
It's ok for me now.
 I have says that before your change to main.js